src/RelayServer.ts

```diff
@@ -214,13 +219,14 @@ export class RelayServer extends EventEmitter {
             );
         }
 
-        // Check the relayWorker (todo: once migrated to multiple relays, check if exists)
+        // Check the feeReceiver (todo: once migrated to multiple relays, check if exists)
         if (
-            req.relayRequest.relayData.relayWorker.toLowerCase() !==
-            this.workerAddress.toLowerCase()
+            [this.collectorAddress, this.workerAddress].includes(
```

Comment:
Im not sure if Im not following it correctly, but, this should be a negation isn't it?

Reply:
right, thanks for the observation